tools/Dockerfile.builder

```diff
@@ -3,9 +3,80 @@
 # as well as in the Developer docs - https://github.com/envoyproxy/gateway/blob/main/DEVELOPER.md#prerequisites
 
 # go
-FROM golang:1.18.2 as builder
+FROM golang:1.18.3 as binary_tools_context
+
+# install binary tools in this context
```

Comment:
imho all this seems excessive at this point
the go image is based on Debian and already contains most tools
https://github.com/docker-library/golang/blob/master/1.18/buster/Dockerfile

the install steps for the linters seem to be a one line `RUN` step which is good

the python install can be a single RUN step,  `apt install python3-pip` is enough to install `pip` and its dependencies

Reply:
That's a good point, I've changed the Dockerfile.builder, PTAL.